.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -25,21 +25,21 @@ body:
   - type: textarea
     id: environment
     attributes:
-      label: Command to run and results
+      label: Command to run and resulting output
       description: "Please paste here what you ran (command, vscode) and what were your results that you think ought to be different:"
       render: code
   - type: textarea
     id: what-happened
     attributes:
       label: What happened?
       description: Describe the issue here. What did you expect instead?
-      placeholder: Tell us what you see!
+      placeholder: Tell us what you wish to see!
```

Comment:
Why?

Reply:
I guess it's both, given that the description ask for both observed and expected behaviour. How about:

> Tell us what you see and wish to see!